scipy/optimize/tests/test_isotonic_regression.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+from numpy.testing import assert_almost_equal, assert_equal
```

Comment:
Sorry for the last minute comment but before merging it would be nice to replace the use of `assert_almost_equal` with the preferred `assert_allclose`

Reply:
Ok, I'm off your toes Jake, the green button is yours when you approve :-)